### ISSUE TITLE
test.py performance metrics and train.py performance metrics

### ISSUE THREAD

**Issue by mjgmjgmjg** [ultralytics/yolov5#12864]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello, I am publishing a paper on Yolov5, I would like to ask if the performance indicators of Yolov5 are evaluated in train.py or test.py? I am using Yolov5 version 5.0

**Comment by mjgmjgmjg** [comment]
Should the performance indicators used for publication be train.py or test.py?

**Comment by glenn-jocher** [comment]
@mjgmjgmjg hi there! 👋 For publishing, you should use the performance metrics generated by `test.py`. This script evaluates the model on a separate test set, providing an unbiased assessment of its performance. `train.py` is primarily for training and provides feedback on the training and validation sets. Always ensure your paper cites these test metrics for a precise evaluation. Happy publishing! 🚀
